{{cookiecutter.project_slug}}/frontend/src/views/main/admin/EditUser.vue

```diff
@@ -1,145 +1,187 @@
-                  v-model="password2"
-                  :error-messages="errors.first('password_confirmation')"
-                >
-                </v-text-field>
-              </v-flex>
-            </v-layout>
-          </v-form>
-        </template>
-      </v-card-text>
-      <v-card-actions>
-        <v-spacer></v-spacer>
-        <v-btn @click="cancel">Cancel</v-btn>
-        <v-btn @click="reset">Reset</v-btn>
-        <v-btn
-          @click="submit"
-          :disabled="!valid"
-        >
-          Save
-        </v-btn>
-      </v-card-actions>
-    </v-card>
+                  v-model="email"
+                  label="E-mail"
+                  type="email"
+                  :error-messages="errors[0]"
+                  required
+                ></v-text-field>
+              </validation-provider>
+
+              <!-- is superuser -->
+              <div class="subheading secondary--text text--lighten-2">
+                User is superuser
+                <span v-if="isSuperuser">(currently is a superuser)</span
+                ><span v-else>(currently is not a superuser)</span>
+              </div>
+              <v-checkbox v-model="isSuperuser" label="Is Superuser"></v-checkbox>
+
+              <!-- is active -->
+              <div class="subheading secondary--text text--lighten-2">
+                User is active <span v-if="isActive">(currently active)</span
+                ><span v-else>(currently not active)</span>
+              </div>
+              <v-checkbox v-model="isActive" label="Is Active"></v-checkbox>
+              <v-row align="center">
+                <v-col class="flex-shrink-1">
+                  <v-checkbox v-model="setPassword" class="mr-2"></v-checkbox>
+                </v-col>
+
+                <v-col>
+                  <!-- password -->
+                  <validation-provider
+                    v-slot="{ errors }"
+                    :debounce="100"
+                    name="Password"
+                    vid="password1"
+                    :rules="{ required: setPassword }"
+                  >
+                    <v-text-field
+                      v-model="password1"
+                      type="password"
+                      :disabled="!setPassword"
+                      label="Password"
+                      :error-messages="errors"
+                    ></v-text-field>
+                  </validation-provider>
+
+                  <!-- password confirmation -->
+                  <validation-provider
+                    v-slot="{ errors }"
+                    :debounce="100"
+                    name="Password confirmation"
+                    vid="password2"
+                    :rules="{
+                      required: setPassword,
+                      confirmed: setPassword ? 'password1' : false,
+                    }"
+                  >
+                    <v-text-field
+                      v-model="password2"
+                      type="password"
```

Comment:
Like in `password1`, would it be nice, if this input field is also disabled if `setPassword` is false.
```vue
                    <v-text-field
                      v-model="password2"
                      type="password"
                      :disabled="!setPassword"
                      label="Confirm Password"
                      :error-messages="errors"
                    ></v-text-field>
```

Reply:
Again, something for the main branch.